builder/bundleset.go

```diff
@@ -54,7 +56,9 @@ func validateAndFillBundleSet(bundles bundleSet) error {
 	}
 	for _, b := range sortedBundles {
 		var allPackages []string
-		allPackages = append(allPackages, b.DirectPackages...)
+		for k := range b.DirectPackages {
```

Comment:
Should this check that the bool associated with k is true or do we assert we will only every remove things using delete rather than setting the value to false?

Reply:
I *was* relying on removing the keys if necessary, but it would be safer and more future-proof to check the bool as well.

Reply:
Done.